topology/probes/netlink.go

```diff
@@ -373,11 +373,14 @@ Loop:
 	for {
 		n, err := syscall.EpollWait(epfd, events[:], 1000)
 		if err != nil {
-			logging.GetLogger().Error("Failed to receive from netlink messages: %s", err.Error())
-			continue
+			errno, ok := err.(syscall.Errno)
+			if ok && errno != syscall.EINTR {
```

Comment:
The idea is to keep the original behavior, except that with a `syscall.EINTR` (Ctrl+C for instance) we don't restart the loop. In this case, `n == -1`, that why I changed `n == 0` to `n < 0`.